_data/jobs.yml

```diff
@@ -1,3 +1,5 @@
+- {expires: 2021-01-06, location: 'University of Alabama, Tuscaloosa, AL (remote work possible)', name: Research Softwaer Engineer, 
```

Comment:
One tiny typo:

```suggestion
- {expires: 2021-01-06, location: 'University of Alabama, Tuscaloosa, AL (remote work possible)', name: Research Software Engineer, 
```

Reply:
Change is good.

Reply:
That's weird - there is some kind of bug when I try to commit the suggestion, it just resolves the entire thing (and doesn't commit). @JeffCarver I think for now you might just need to push again with the fix, sorry for the annoyance!